classes/Logs.php

```
@@ -170,42 +58,59 @@ public static function clearLogs(): bool
 
-        $args = [
-            'post_type' => self::postType(),
-            'posts_per_page' => -1,
-            'post_status' => 'any',
-        ];
-
-        $posts = get_posts($args);
-
-        foreach ($posts as $post) {
-            wp_delete_post($post->ID, true);
+        $files = glob(MAWIBLAH_LOG_PATH . 'mawiblah-*.log');
```

PHPStan (2.2.2)

[error] Constant MAWIBLAH_LOG_PATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)